### ISSUE TITLE
Object recognition in images

### ISSUE THREAD

**Issue by Sujithavemban** [AldoStalin/SonaITHackathon#37]
Stars -Object recognition in images
## ℹ️ Project information
_Please complete all applicable._
- **Project Name**:Object recognition in images
- **Short Project Description**: In our project we predict the objects from the given image using CNN Algorithm. 
- **Team Name**:Stars
- **Team Members**: Sujitha.V-@Sujithavemban & Sindhuja.S-@SSDsindhuja
- **Demo Link**:https://youtu.be/GayX3FHZGRE
- **Repository Link**:https://github.com/Sujithavemban/Object-recognition-in-images.git
- **Labels**: "Machine Learning","Robotics Application"

## 🔥 Your Pitch:
 Object recognition in images is challenging task in the area of computer vision and image processing in this work we present object classification and detection using our own created dataset.we used convolutional neural network on keras with tensorflow support the experimental results shows the time required to train, test and create the model in limited computing system. CNNs scan the image with learnable “filters” and extract more and more abstract features at each layer. Filters in early layers may for example detect edges or color gradients, while later layers may register complex shapes.
## 🔦 highlight:Using CNN Algorithm predict the output.

**Before you post the issue**:
- ✅You have followed the issue title format.
- ✅ You have applied the correct labels.
- ✅ You have mentioned the correct labels.
- ✅ You have provided all the information correctly.
